packages/apex-node/package.json

```diff
@@ -6,7 +6,7 @@
   "bugs": "https://github.com/forcedotcom/salesforcedx-apex/issues",
   "main": "lib/src/index.js",
   "dependencies": {
-    "@salesforce/core": "2.33.1",
+    "@salesforce/core": "^2.35.0",
```

Comment:
This is how it is specified in SDR so I am attempt to sync that across all three projects.